package.json

```diff
@@ -94,7 +97,7 @@
     "sinon": "^6.1.3",
     "sinon-chai": "^3.2.0",
     "style-loader": "^0.21.0",
-    "supertest": "^3.1.0",
```

Comment:
so turns out that my test failures in omic were due to an old version of supertest :angry: